apps/prairielearn/elements/pl-dataframe/pl-dataframe.py

```diff
@@ -174,9 +174,24 @@ def render(element_html: str, data: pl.QuestionData) -> str:
     # Might be worth moving everything out of the CSS file and handle it all with the builtin styler.
     frame_style.set_table_attributes("class=pl-dataframe-table")
 
+    # Generate the initial HTML
+    frame_html_raw = frame_style.to_html()
+
+    # https://www.w3.org/WAI/WCAG21/Techniques/html/H63
+    html_root = lxml.html.fromstring(frame_html_raw)
+    for th in html_root.xpath("//th[contains(@class, 'blank')]"):
+        th.set("scope", "colgroup")
```

Comment:
Which of the examples reproduces something with a `th.blank`?

Reply:
![CleanShot 2025-04-24 at 13 10 52@2x](https://github.com/user-attachments/assets/90e93110-a9a4-451d-9b73-8c51c2e547a3)
`element/codeDocumentation`

Reply:
This line was removed in 062f362